### PR TITLE
Working tree encoding

### PR DESCRIPTION
**Pull request by schwarherz**
Adds working-tree-encoding to the repository to keep the formatting in Windows-1252 when downloaded from the repository (stored in UTF-8 on the website)

### CODE REVIEW

File: CleanSlate/events/base_rumours_events.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@
 ###################################
 #
 # Designed by Johan Andersson
-# Written by Sara Wendel-Örtqvist
+# Written by Sara Wendel-Ã–rtqvist
```
Comment (unique-usernames-are-annoying) [inline CleanSlate/events/base_rumours_events.txt:11]: Ã–rtqvist

File: CleanSlate/events/sunset_invasion.txt [hunk 2]
```diff
@@ -5,7 +5,7 @@
 #
 ###################################
 
-# Written by Henrik F�hraeus and Henrik Eklund
+# Written by Henrik Fï¿½hraeus and Henrik Eklund
```
Comment (unique-usernames-are-annoying) [inline CleanSlate/events/sunset_invasion.txt:8]: Fï¿½hraeus

File: CleanSlate/history/titles/c_northumberland.txt [hunk 3]
```diff
@@ -40,7 +40,7 @@
 }
 
 568.1.1 = {
-	holder = 205205 # ∆thelric
+	holder = 205205 # âˆ†thelric
```
Comment (unique-usernames-are-annoying) [inline CleanSlate/history/titles/c_northumberland.txt:43]: âˆ†thelric

File: CleanSlate/events/base_rumours_events.txt (hunk 1 above)
Comment (schwarherz) [reply CleanSlate/events/base_rumours_events.txt:11]:
```suggestion
# Written by Sara Wendel-Örtqvist
```

File: CleanSlate/events/sunset_invasion.txt (hunk 2 above)
Comment (schwarherz) [reply CleanSlate/events/sunset_invasion.txt:8]:
```suggestion
# Written by Henrik Fåhraeus and Henrik Eklund
```

File: CleanSlate/history/titles/c_northumberland.txt (hunk 3 above)
Comment (schwarherz) [reply CleanSlate/history/titles/c_northumberland.txt:43]:
```suggestion
	holder = 205205 # Æthelric
```

File: CleanSlate/common/modifier_definitions/_documentation.info [hunk 4]
```diff
@@ -39,6 +39,6 @@ The name of this token is defined in the offmap itself. Gives monthly offmap cur
 
 == Localisation ==
 The key of the modifier will be used for localisation to show in modifier breakdowns. Example:
-horse_archers_OFFENSIVE_FIRE;Horse Archers Attack;Archers montés – Attaque;Ber. Bogenschützen - Angriff;;Ataque de arqueros montados;;;;;;;;;x
+horse_archers_OFFENSIVE_FIRE;Horse Archers Attack;Archers montÃ©s â€“ Attaque;Ber. BogenschÃ¼tzen - Angriff;;Ataque de arqueros montados;;;;;;;;;x
```
Comment (unique-usernames-are-annoying) [inline CleanSlate/common/modifier_definitions/_documentation.info:42]: montÃ©s â€“ and BogenschÃ¼tzen. Generally all the obviously wrong characters like ©, €, ¼ are likely going to exist a lot more often and should be searched for in all files and similarly Ã, though that will give a lot of correct results as well. You also may want to consider splitting up the huge commit into smaller parts, which might be easier to review individually. At the very least one for txt and another for csv files.

File: CleanSlate/common/offmap_powers/offmap_powers.info [hunk 5]
```diff
@@ -148,14 +148,14 @@ any_character/random_character does not include offmap rulers, but does include
 	has_policy = policy_name - Checks if the offmap has the given policy
 	prev_policy = policy_name - Checks if the offmap had the given policy as its last policy
 	has_policy_flag = flag_name - Checks if the offmap has a policy with the given flag
-	prev_policy_flag = flag_name - Checks if the offmapæs last policy has the given flag
+	prev_policy_flag = flag_name - Checks if the offmapÃ¦s last policy has the given flag
```
Comment (unique-usernames-are-annoying) [inline CleanSlate/common/offmap_powers/offmap_powers.info]: offmapÃ¦s

File: CleanSlate/common/offmap_powers/offmap_powers.info [hunk 6]
```diff
@@ -148,14 +148,14 @@ any_character/random_character does not include offmap rulers, but does include
...
 	days_since_policy_change > 1 - Checks how long ago the policy changed. Note that setting the same policy as is currently present (E.G., "changing" to china_stable while china_stable is already active) will reset the timer
 
 # status related triggers
 	has_status = status_name - Checks if the offmap has the given status
 	prev_status = status_name - Checks if the offmap had the given status as its last status
 	has_status_flag = flag_name - Checks if the offmap has a status with the given flag
-	prev_status_flag = flag_name - Checks if the offmapæs last status has the given flag
+	prev_status_flag = flag_name - Checks if the offmapÃ¦s last status has the given flag
```
Comment (unique-usernames-are-annoying) [inline CleanSlate/common/offmap_powers/offmap_powers.info:158]: offmapÃ¦s

**Comment by schwarherz** [comment]
Closing without merging. Will make another branch for this change later when all outstanding changes are in-place.

**Pull request closed without merging**